localflavor/br/forms.py

```diff
@@ -11,12 +11,10 @@
 from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
 
+from localflavor.br import validators
```

Comment:
For consistency, can you use relative imports like this: `from .validators import postal_code_re, ...`. This applies to other files as well.